app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -8,6 +8,9 @@ module Resolution
     class ProgressiveProofer
       # @param [Hash] applicant_pii keys are symbols and values are strings, confidential user info
       # @param [Boolean] double_address_verification flag that indicates if user will have
+      #   both state id address and current residential address verified. Note this value is here as
+      #   a placeholder until it can be replaced with ipp_enrollment_in_progress
```

Comment:
can we add the ticket number that will address this? (the new one you added)

Reply:
Yep! I made the ticket last night. I'll get that noted in this comment now. Great idea @svalexander